ccf_transaction_processor/Makefile

```diff
@@ -77,8 +83,7 @@ clean-install :
 config : $(ETCDIR)/cchost.toml $(ETCDIR)/gov.lua
 
 $(ETCDIR)/cchost.toml : etc/cchost.toml
-	@ echo create configuration files for cchost
-	@ . $(abspath $(DSTDIR)/bin/activate) ; \
+	@ . $(abspath $(CCFDSTDIR)/bin/activate) ; \
 		$(CNFGEN) --template $(notdir $<) --template-directory $(dir $<) \
```

Comment:
We really should use CCFDSTDIR everywhere it is appropriate (see the macros at the top of the file for example). This is not a show stopper especially since i'm sure we'll keep revising this makefile for awhlie.

Reply:
my patch i've sent to Prakash with which i could make it work actully did that ;-)

Reply:
applied patch, Thank you!